make/docker.mk

```diff
@@ -0,0 +1,27 @@
+##@ Cli Development
+
+# ================================================
+# Public Commands:
+# ================================================
+
+.PHONY: docker-compose-up
+docker-compose-up: ## Run docker-compose up -d
+docker-compose-up: docker.compose.up
+
+.PHONY: docker-compose-down
+docker-compose-down: ## Run docker-compose down
+docker-compose-down: docker.compose.down
+
+# ================================================
+# Private Commands:
+# ================================================
+
+.PHONY: docker.compose.up
+docker.compose.up:
+	@echo "======> Running Docker compose up"
+	@docker-compose -f docker/service/docker-compose.yaml up -d
```

Comment:
this command run with `--build` option is better, which can rerun `go build` when the codes has modified.

Reply:
fixed it.